src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -1439,6 +1439,7 @@ protected void free(Clob clob) throws SQLException {
   }
 
   @Override
+  @SuppressWarnings("deprecation")
```

Comment:
Can these be migrated to the new signature before the method is removed, or do they need to stay while the old signature is deprecated?

Reply:
What do you mean by "these"?

This is mostly a signal to developers to implement the newer method, and IMO there's not really a good reason to remove these old methods (the earliest we could do that is in a future 7.0.0 release).